pkg/init/Dockerfile

```diff
@@ -1,3 +1,9 @@
+FROM alpine:edge AS build
```

Comment:
when I use `-p out`, `RUN` still doesn't pick up the commands; I dropped that, which I think is fine since this is just a stage in a multi-stage build, so it won't bloat the final image.